### ISSUE TITLE
Single file input is ignored when using CLI

### ISSUE THREAD

**Issue by huyph** [stylelint/stylelint#2204]
A bug where running this seems to work, it generates some errors:
```
$ cat test.css | stylelint -f verbose

<input css 1>
 2:5   ✖  Expected indentation of 2 spaces   indentation 
 2:12  ✖  Unexpected hex color "#123456aa"   color-no-hex

1 source checked
 <input css 1>

2 problems found
 severity level "error": 2
  color-no-hex: 1
  indentation: 1
```

But, this does NOT work
```
$ stylelint "test.css" -f verbose

0 of 1 source checked
 test.css (ignored)

0 problems found
```

> What CSS is needed to reproduce this issue?
```css
a {
    color: #123456aa;
}
```

> What stylelint configuration is needed to reproduce this issue?
```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "color-no-hex": [true, { "severity": "error" } ]
  }
}
```

> Which version of stylelint are you using?
`7.7.0`

> What did you expect to happen?
Both commands above should generate the same outcome

**Comment by jeddy3** [comment]
@huyph Thank you for the report and using the issue template.

I can't reproduce this. The following:

```
stylelint "test.css" -f verbose
```

Give me:

```
test.css
 2:5   ✖  Expected indentation of 2 spaces   indentation
 2:12  ✖  Unexpected hex color "#123456aa"   color-no-hex

1 source checked
 /Users/jeddy3/Projects/stylelint-test/test.css

2 problems found
 severity level "error": 2
  color-no-hex: 1
  indentation: 1
```

What directory is your `test.css` located? Is it within a `node_modules` directory?

If not can you create a reproducible test repo please to help us get to the bottom of this.

**Comment by davidtheclark** [comment]
@huyph: Why is the file "ignored"? What are your ignore settings, in `.stylelintrc` or `.stylelintignore`?

**Comment by huyph** [comment]
Thanks for getting back quickly. 👍 

I think I have found the issue. In my setup, I have `test.css` in a package inside node_modules. If I add "stylelint" to the list of scripts in that package's package.json then running `npm run stylelint tmp.css` works. 

Closing this ticket now.
